### ISSUE TITLE
Recording wrapper, mirror input wrapper ignore shift + arrow

### ISSUE THREAD

**Issue by KatieWoe** [phetsims/gravity-force-lab#245]
For https://github.com/phetsims/QA/issues/471. Happens in both chrome and firefox so far. Similar to https://github.com/phetsims/gravity-force-lab/issues/242.
If you use keyboard commands to move the robots or the mass sliders, and use shift+arrow to move in small increments, then the top sim will behave as expected, but the bottom one will act as if there was no shift key.
![noshiftarro](https://user-images.githubusercontent.com/41024075/73570966-b8ebd580-442a-11ea-9652-d19a8ac78f30.gif)

**Comment by loganbraywork** [comment]
This also appears to occur in the recording wrapper, giving an incorrect playback when shift is used to move either the slider or masses by small increments.

**Comment by KatieWoe** [comment]
Applies to metacog recording too

**Comment by zepumph** [comment]
This was as simple as serializing the modifier keys in a dom event. This way the incoming input event to the downstream sim (without the context of the actual shift key down event), can still know that the shift key is down. `Input.js` takes care of the rest. 

@jonathanolson please review. Does this seem like a reasonable solution to you? Can you think of any non phet-io/a11y use cases for `Input.serializeDOMEvent` that this will cause trouble for?

**Comment by zepumph** [comment]
@jonathanolson and I reviewed this and it looks good. I will take one more look through the MDN docs for window.Event to see if we are missing any others.

**Comment by zepumph** [comment]
I added `type` for clarity, though it doesn't need to be deserialized. Ready to be cherry-picked.

**Comment by loganbraywork** [comment]
Appears fixed
